Src/AutoFakeItEasy2.UnitTest/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("AutoFakeItEasyUnitTest")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Ploeh")]
+[assembly: AssemblyProduct("AutoFixture")]
+[assembly: AssemblyCopyright("Copyright © Ploeh 2011")]
```

Comment:
I don't think we can claim that the copyright of this particular library stretches back to 2011; it ought to be 2016. Also, I think it'd be fairer to assign the copyright to _AutoFixture_, although that isn't a legal entity...